package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "bbc-a11y",
-  "version": "3.0.0",
+  "version": "2.3.0",
```

Comment:
Please do not edit the bbc-a11y version number when submitting a commit. This will be done as part of the release process.